test/JDBC/jdbc_schedule

```diff
@@ -10,6 +10,8 @@
 
 all
 
+ignore#!#BABEL-SP_DATABASES
```

Comment:
please remove this and comment out particular test case. Not whole test file.

Reply:
done